AUTHORS.txt

```diff
@@ -0,0 +1,16 @@
+Michele Ceriotti
+Félix Musil
+Max Veit
+Alexander Goscinski
```

Comment:
Might be better to sort this file by some alphabetic order? I'm not sure how you generated it =)

Reply:
had some idea behind it, but it seems a bit random. changed to alphabetic